software-modules/debuggers/valgrind/valgrind.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+#	vim.sh
+#	Script to build the modular software package of Vim
+#	for huronOS. It purges the unnecessary files on the FS
+#	to allow AUFS add/del operations on the fly.
+#
+#	Copyright (C) 2022, huronOS Project:
+#		<http://huronos.org>
+#
+#	Licensed under the GNU GPL Version 2
+#		<http://www.gnu.org/licenses/gpl-2.0.html>
+#
+#	Authors:
+#		Enya Quetzalli <equetzal@huronos.org>
+
+set -xe
+NAME=valgrind
+
+## Install software
+apt update
+apt install --yes --no-install-recommends $NAME
+apt autoremove --yes
+
+## Prepare final files
+cp ./$NAME.desktop /usr/share/applications/
```

Comment:
@DT3264 Does this file exists? (I mean, was missing on the PR) or this line was not supposed to exist?

Reply:
It was supossed to exists, I might have done something to accidentally erase it, will fix it rn